packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-foot-note.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { getAuthenticationStatusInfo } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import { TJurisdictionModalFootNoteProps } from '../props.types';
+
+const JurisdictionModalFootNote = ({
+    account_status,
+    account_type,
+    jurisdiction_selected_shortcode,
+    card_classname,
+}: TJurisdictionModalFootNoteProps) => {
+    const {
+        poi_or_poa_not_submitted,
+        poi_pending_for_vanuatu,
+        poi_verified_for_bvi_labuan_maltainvest,
+        poi_pending_for_bvi_labuan_maltainvest,
+        poi_verified_for_vanuatu,
+        poi_not_submitted_for_vanuatu,
+    } = getAuthenticationStatusInfo(account_status);
+
+    const account_type_name = account_type && (account_type === 'synthetic' ? 'Synthetics' : 'Financial');
```

Comment:
```suggestion
```
This is unnecessary. You can capitalise the first letter using CSS at the `Text` component 

`text-transform: capitalize`